contributor/Development_Workflow.md

```diff
@@ -0,0 +1,26 @@
+# Development Workflow
+
+## Prerequisites
+The OpenLab CI is built based on `NodePool` and `Zuul` tools of the OpenStack infrastructure. The CI Jobs definition uses Ansible playbooks. Learn more about the tools first:
+* Zuul documentation is hosted on [Zuul](https://zuul-ci.org)
+* NodePool documentation is hosted on [NodePool](https://zuul-ci.org/docs/nodepool/)
+* Ansible documentation is hosted on [Ansible Docs](https://docs.ansible.com/)
+
+## Working on Ocean request
+The accepted test request items can be found on [TODO List](https://github.com/orgs/theopenlab/projects/1#column-1860008).
+1. Pick one item you wish to work on, move it to [IN PROGRESS List](https://github.com/orgs/theopenlab/projects/1#column-1860011), edit the item to add the development plan
```

Comment:
Pike an item you want to work on, and move it to the **[IN PROGRESS](https://github.com/orgs/theopenlab/projects/1#column-1860011)** list, you can also feel free to edit the item according to your development plan.

Reply:
s/one item/one test request/